src/commands/ecr-login.yml

```
@@ -116,10 +116,10 @@ steps:
         - run:
             name: Log into Amazon ECR with profile <<parameters.new-profile-name>>
             environment:
-              PARAM_PROFILE_NAME: <<parameters.new-profile-name>>
-              PARAM_REGISTRY_ID: <<parameters.registry-id>>
-              PARAM_REGION: <<parameters.region>>
-              PARAM_PUBLIC_REGISTRY: <<parameters.public-registry>>
+              ORB_VAL_PROFILE_NAME: <<parameters.new-profile-name>>
+              ORB_VAL_REGISTRY_ID: <<parameters.registry-id>>
```

```suggestion
              ORB_ENV_REGISTRY_ID: <<parameters.registry-id>>
```

For parameters type of `env_var_name` the prefix should be `ORB_ENV_`. This indicates later on that you should indirectly reference the environment variable.